Lombiq.BaseTheme.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -39,6 +39,6 @@ public static async Task TestMainMenuWithAuthenticationAsync(this UITestContext
         await context.ClickMainMenuPathAsync("Account", "Log Out");
 
         await context.ClickMainMenuPathAsync("Log In");
-        context.Exists(By.XPath("//form[@action = '/Login']/h4[contains(., 'Log in')]"));
+        context.Exists(By.XPath("//form[@action = '/Login']/h1[contains(., 'Log in')]"));
```

Comment:
I don't think the header level matters here so we can generalize:
```suggestion
        context.Exists(By.XPath("//form[@action = '/Login']/*[starts-with(name(), 'H' and [contains(., 'Log in')]"));
```